src/sentry/conf/server.py

```diff
@@ -1749,15 +1752,22 @@ def build_cdc_postgres_init_db_volume(settings):
     ),
     "zookeeper": lambda settings, options: (
         {
-            "image": "confluentinc/cp-zookeeper:5.1.2",
+            # On Apple arm64, we upgrade to version 6.x allows zookeeper to run properly on Apple's arm64
+            # See details https://github.com/confluentinc/kafka-images/issues/80#issuecomment-855511438
+            "image": "confluentinc/cp-zookeeper:6.2.0"
+            if APPLE_ARM64
+            else "confluentinc/cp-zookeeper:5.1.2",
```

Comment:
I'm selectively upgrading the version on Apple's arm64 since there was [a bug that only affects Intel](https://github.com/getsentry/sentry/pull/28574#issuecomment-922087892) (You can read more details in #28672).

I've added a card to investigate this later on.

Reply:
Added